geode-core/src/main/java/org/apache/geode/cache/Region.java

```diff
@@ -699,7 +699,9 @@ void loadSnapshot(InputStream inputStream)
   V put(K key, V value, Object aCallbackArgument) throws TimeoutException, CacheWriterException;
 
   /**
-   * Creates a new entry in this region with the specified key and value.
+   * Creates a new entry in this region with the specified key and value if and only if an entry
+   * does not already exist for the specified key. If an entry already exists for the specified
+   * key, throws {@link EntryExistsException}.
```

Comment:
It looks like we already have an @throws for this exception in this javadoc.  I don't think we need to spell it out twice.

Reply:
Perhaps not for smart people. But it wasn't immediately obviously **to me** that it throws an exception when the entry already exists. @upthewaterspout knew that already and pointed me to the list of exceptions. But I'm not sure that everyone reads the {{@throws}} declarations and I think only the first sentence of the description is shown in the table of a class' methods.